Submitted By: Bruce Dubbs <bdubbs at linuxfromscratch dot org>
Date: 2018-03-30
Initial Package Version: 2.89
Upstream Status: Not Submitted
Origin: Self
Description: Remove programs superceeded by other packages: wall, mountpoint,
             last, lastb, logsave, mesg, sulogin, and utmpdump.

Rediffed for version 3.13.

diff -Naur sysvinit-3.12.orig/src/Makefile sysvinit-3.12/src/Makefile
--- sysvinit-3.12.orig/src/Makefile	2024-12-21 13:31:09.000000000 -0600
+++ sysvinit-3.12/src/Makefile	2024-12-30 12:30:05.842565325 -0600
@@ -23,19 +23,22 @@
 
 # For some known distributions we do not build all programs, otherwise we do.
 BIN	=
-SBIN	= init halt shutdown runlevel killall5 fstab-decode logsave
-USRBIN	= last mesg readbootlog
+SBIN	= init halt shutdown runlevel killall5 fstab-decode
+#USRBIN	= last mesg readbootlog
 
-MAN1	= last.1 lastb.1 mesg.1 readbootlog.1
+#MAN1	= last.1 lastb.1 mesg.1 readbootlog.1
 MAN5	= initscript.5 inittab.5 initctl.5
-MAN8	= halt.8 init.8 killall5.8 pidof.8 poweroff.8 reboot.8 runlevel.8
-MAN8	+= shutdown.8 telinit.8 fstab-decode.8 logsave.8
+#MAN8	= halt.8 init.8 killall5.8 pidof.8 poweroff.8 reboot.8 runlevel.8
+MAN8 = halt.8 init.8 killall5.8         poweroff.8 reboot.8 runlevel.8
+MAN8	+= shutdown.8 telinit.8 fstab-decode.8
 
 ifeq ($(DISTRO),)
-SBIN	+= sulogin bootlogd
-USRBIN	+= utmpdump wall
-MAN1	+= utmpdump.1 wall.1
-MAN8	+= sulogin.8 bootlogd.8
+#SBIN	+= sulogin bootlogd
+SBIN	+= bootlogd
+#USRBIN	+= utmpdump wall
+#MAN1	+= utmpdump.1 wall.1
+#MAN8	+= sulogin.8 bootlogd.8
+MAN8	+= bootlogd.8
 endif
 
 ifeq ($(DISTRO),Debian)
@@ -216,10 +219,10 @@
 		ln -sf halt $(ROOT)$(base_sbindir)/reboot
 		ln -sf halt $(ROOT)$(base_sbindir)/poweroff
 		ln -sf init $(ROOT)$(base_sbindir)/telinit
-		ln -sf $(base_sbindir)/killall5 $(ROOT)$(base_bindir)/pidof
-		if [ ! -f $(ROOT)$(bindir)/lastb ]; then \
-			ln -sf last $(ROOT)$(bindir)/lastb; \
-		fi
+		#ln -sf $(base_sbindir)/killall5 $(ROOT)$(base_bindir)/pidof
+		#if [ ! -f $(ROOT)$(bindir)/lastb ]; then \
+		#	ln -sf last $(ROOT)$(bindir)/lastb; \
+		#fi
 		$(INSTALL_DIR) $(ROOT)$(includedir)/
 		$(INSTALL_DATA) initreq.h $(ROOT)$(includedir)/
 		for lang in  '' $(patsubst ../man/po/%.po,%,$(wildcard ../man/po/??.po)); do \
@@ -243,8 +246,8 @@
 		#
 		# This part is skipped on Debian systems, the
 		# debian.preinst script takes care of it.
-		@if [ ! -p /run/initctl ]; then \
-		 echo "Creating /run/initctl"; \
-		 rm -f /run/initctl; \
-		 mknod -m 600 /run/initctl p; fi
+		#@if [ ! -p /run/initctl ]; then \
+		# echo "Creating /run/initctl"; \
+		# rm -f /run/initctl; \
+		# mknod -m 600 /run/initctl p; fi
 endif
